src/services/city_services/get_city_by_id.js

```diff
@@ -0,0 +1,20 @@
+class GetCityById {
+    constructor({ cityDaos }) {
+        this.cityDaos = cityDaos;
+
+        this.execute = this.execute.bind(this);
+    }
+
+    async execute(params) {
+        try {
+            const { id } = params;
+            const daosResult = await this.cityDaos.getById(id);
+            if (daosResult.failure || !daosResult.length) throw new Error(daosResult.message || "City is not found");
```

Comment:
m test trường hợp `"City is not found"` của đoạn code này chưa?

Reply:
uhm t test rồi, ổn á